sound/soc/sof/ipc4-topology.c

```diff
@@ -1040,6 +1068,9 @@ static void sof_ipc4_unprepare_copier_module(struct snd_sof_widget *swidget)
 			struct sof_ipc4_alh_configuration_blob *blob;
 			unsigned int group_id;
 
+			if (pipeline->use_chain_dma)
```

Comment:
do we need to use chain dma for "ipc4_copier->dai_type == SOF_DAI_INTEL_ALH"

Reply:
no we dont. ALH uses the GPDMA

Reply:
Oh, so this "if (pipeline->use_chain_dma)" is actually badly placed in side "if (ipc4_copier->dai_type == SOF_DAI_INTEL_ALH)". I'll move it outside.